flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionGraphTestUtils.java

```diff
@@ -398,6 +401,10 @@ public static JobVertex createJobVertex(String task1, int numTasks, Class<NoOpIn
 		return groupVertex;
 	}
 
+	public static ExecutionJobVertex getExecutionVertex(JobVertexID id) throws Exception {
```

Comment:
I know that ExecutionGraphTestUtils did have several `getExecutionVertex (...)` methods to map `JobVertexID` to `ExecutionJobVertex `, but I think it's better we rename the old and newly added ones to `getExecutionJobVertex (...)`. Because `ExecutionVertex` is a structure different from `ExecutionJobVertex`.

Reply:
Yeah, I think that a good idea to make the naming consistent. 

However, there might be too many places to change, and might be good to open a separate PR for this purpose.

Reply:
I did a try and seems the change is not that large.
I'm fine you keep it as is for now since it is out of the scope of this PR. I can apply a patch to rename it later.